base/sparse/linalg.jl

```diff
@@ -499,7 +499,196 @@ function norm(A::SparseMatrixCSC,p::Real=2)
     throw(ArgumentError("invalid p-norm p=$p. Valid: 1, Inf"))
 end
 
-# TODO
+# TODO rank
+
+# cond
+function cond(A::SparseMatrixCSC, p::Real=1)
```

Comment:
@tkelman I agree, this bugs me as well. Still, defaulting to a non implemented feature `p=2` would be rather annoying.  As an alternative, we could enforce the use of both arguments when `A` is sparse. 

You're right, those blank lines aren't that useful. I'll delete them next time I amend the commit.